src/js/libgeo/GeoOps.js

```diff
@@ -1226,6 +1226,46 @@ geoOps.ConicBy2Foci1P.updatePosition = function(el) {
 
 };
 
+// Given (A, a, B, b, C), compute conic such that
+// 1. (A, a) and (B, b) are pole-polar pairs and
+// 2. C is incident with the conic
+geoOps.ConicBy2Pol1P = {};
+geoOps.ConicBy2Pol1P.kind = "C";
+geoOps.ConicBy2Pol1P.signature = ["P", "L", "P", "L", "P"];
+geoOps.ConicBy2Pol1P.updatePosition = function(el) {
+    var A = csgeo.csnames[(el.args[0])].homog;
+    var a = csgeo.csnames[(el.args[1])].homog;
+    var B = csgeo.csnames[(el.args[2])].homog;
+    var b = csgeo.csnames[(el.args[3])].homog;
+    var C = csgeo.csnames[(el.args[4])].homog;
+
+    var sp = List.scalproduct;
+    var sm = List.scalmult;
+    var sub = List.sub;
+    var rm = CSNumber.realmult;
+    var dc = geoOps._helper.degeneratePairOfJoiningLines;
```

Comment:
Applause for a very fine result indeed!
Here the dc variable goes unused an can be removed:
```javascript
    //var dc = geoOps._helper.degeneratePairOfJoiningLines; 
```